examples/example_GUI.py

```diff
@@ -379,17 +381,34 @@ def _transform(self):
                 self.text.insert(
                    tk.END,
                    "\ndirect: calculation is slowed if Cython unavailable ...")
+            self.text.see(tk.END)
             self.canvas.draw()
 
-            if self.method == 'linbasex':
-                self.AIM = abel.Transform(
-                    self.IM, method=self.method, direction=self.fi,
-                    transform_options=dict(return_Beta=True))
-            else:
-                self.AIM = abel.Transform(
-                    self.IM, method=self.method, direction=self.fi,
-                    transform_options=dict(basis_dir='bases'),
-                    symmetry_axis=None)
+            try:
+                if self.method == 'linbasex':
+                    self.AIM = abel.Transform(
+                        self.IM, method=self.method, direction=self.fi,
+                        transform_options=dict(return_Beta=True))
+                else:
+                    basis_dir = 'bases'
+                    if not os.path.isdir(basis_dir):
```

Comment:
I agree!
I can add a check hard-coded there or, to keep things more localized and maintainable, define something like `Abel_methods_basis = [...]` where `Abel_methods` are defined and then check for `self.method in Abel_methods_basis`.

It might be a good idea to also define `Abel_methods_forward = [...]`, since not all methods support the forward transform. Now that `Transform()` errors are reported in the GUI, such failures are at least visible, but it would be better to disable “forward” when it is not supported.

Reply:
`Abel_methods_basis` and `Abel_methods_forward` are a good idea.

Reply:
Done!